documentation/modules/metrics/proc_metrics-deploying-grafana.adoc

```diff
@@ -6,7 +6,7 @@
 
 = Deploying Grafana
 
-To deploy Grafana to provide visualizations of Prometheus metrics, apply the xref:ref-metrics-config-files-{context}[example configuration file].
+To deploy Grafana to provide visualizations of Prometheus metrics, apply the `grafana.yaml` file provided in the `examples/metrics` directory.
```

Comment:
Maybe we should make it more clear that this is just one of the options and users should feel free to use their own Grafana installations?

Reply:
Which should be the same for the Prometheus installation as well.

Reply:
To provide visualizations of Prometheus metrics,
you can use your own Grafana installation or deploy Grafana by applying the `grafana.yaml` file provided in the `examples/metrics` directory.

To obtain monitoring data from your Kafka cluster, you can use your own Prometheus deployment or deploy Prometheus by applying the xref:ref-metrics-config-files-{context}[example resource file for the Prometheus docker image and the YAML files for Prometheus-related resources].